canonical-web-and-design/jaas-dashboard
# Add relation icons

## Done

Added icons to the model details relations table and tweaked some of the columns to better fit the real data.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- View a model that has relations to make sure that the icons for the applications show up.

## Details

Fixes #150

## Screenshots

<img width="1269" alt="Screen Shot 2019-12-17 at 3 34 18 PM" src="https://user-images.githubusercontent.com/532033/71036152-b5062d00-20e2-11ea-9134-8462b7e6e0cc.png">
